### ISSUE TITLE
Remove Control - automatic NVDA restart

### ISSUE THREAD

**Issue by tapintapin** [NVDARemote/NVDARemote#45]
/Hi,

I'm writing to you regarding an issue about restarting NVDA from remote computer.

The situation is that NVDA sometimes gets frozen and won't start up again when controlling it from another computer which causes that you are unable to resume the remote control over a specific computer. When you're away for a few days it's quite bummer as you're dependant on NVDA running and when for some reason NVDA won't start, you can't control the computer anymore before you return home:).

Of course I have the latest NVDA and -r atribute in the desktop icon but sometimes this happens. I thought is it possible to add some commands to the addon which would restart the NVDA in time intervals defined by user? The idea is that when NVDA gets frozen and won't start you would  only wait until next restart at the periodical time you defined e.g. every 8 hours or so.

I've tried system things such as

schtasks /create /sc daily /sd 10/08/2015 /st 09:51 /tn "NVDA_start" /tr "%programfiles(x86)%\NVDA\nvda.exe"
but NVDA didn't get started at that date and time. Also another command
at 10:51 /every:10 "%programfiles(x86)%\nvda\nvda.exe"
but NVDA also didn't get started. I've also tried the absolute pathes c:\ with no success.

Can you guys please give me an advise how to preventively restart NVDA at specific date/time?

Thanks so much for your response
have a nice day
Tapin
